VideoLocker/src/androidTest/java/org/edx/mobile/test/screenshot/EdXScreenshotUtils.java

```diff
@@ -0,0 +1,23 @@
+package org.edx.mobile.test.screenshot;
+
+import android.view.View;
+
+import com.facebook.testing.screenshot.ViewHelpers;
+
+/**
+ * Created by cleeedx on 4/4/16.
+ */
+public class EdXScreenshotUtils {
+
+    /**
+     * Sets the screenshot size to 480x800 which is is bigger than ~92.5% of all screen sizes.
+     * https://developer.android.com/about/dashboards/index.html#Screens
+     */
+    public static void setScreenSizeToNormalHDPI(View view) {
```

Comment:
Should we just call this "defaultTestResolution" or something like that? A client of the call doesn't care that it's HDPI, right?